.travis.yml

```diff
@@ -41,6 +39,8 @@ install:
 
 script:
   - make test
+  - sudo apt-get install python-pip
+  - sudo pip install pylint
   - jdk_switcher use oraclejdk8
   - |-
     PYTHONPATH=libexec/python/ pylint libexec/python/ -r n --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" > pylint-report.txt
```

Comment:
stupid question probably - how/why can you put PYTHONPATH=etc. on the same line without an && or ; to end the line?